src/App.tsx

```diff
@@ -1,14 +1,68 @@
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import debounce from 'lodash.debounce';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { PeopleList } from './components/PeopleList/PeopleList';
+
+const DELAY = 1000;
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [inputText, setInputText] = useState<string>('');
+  const [applyedQuery, setApplyedQuery] = useState('');
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+  const [showSuggestions, setShowSuggestions] = useState<boolean>(false);
```

Comment:
```suggestion
  const [inputText, setInputText] = useState('');
  const [applyedQuery, setApplyedQuery] = useState('');
  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
  const [showSuggestions, setShowSuggestions] = useState(false);
```
it's redundant to specify primitive types